### DISCUSSION TITLE
A request for assistance with the Mobi model simulation of the interactions between macromolecular drugs and small molecule drugs

### DISCUSSION THREAD

**Discussion by Hikari1118** [Open-Systems-Pharmacology/Forum#1769] [Questions & Problems]
I used PK-Sim to simulate a physiologically-based pharmacokinetic (PBPK) model for the interaction between a macromolecule and a small molecule. Then, I exported it to MoBi software to create a PBPK model for pregnant women. After adding the organs specific to pregnant women, imitating the organ compartments of a normal person, I planned to run the model simulation. However, it resulted in an error, and the simulation failed. Here is the table of error logs that I exported. If anyone has experience with this issue and could assist in resolving it, I would greatly appreciate it.
[errors.csv](https://github.com/Open-Systems-Pharmacology/Forum/files/15497584/errors.csv)

**Comment by StephanSchaller** [comment]
It is a lot of errors, so sth. went clearly wrong.
Did you follow an instruction to add the additional organs?
You can check the pregnancy training on our platform or the one on "Tumor treatment" (which showcases the adding one ONE organ in the large molecule model): [learn.esqlabs.com](https://learn.esqlabs.com)

**Comment by wilbertdew** [comment]
@Hikari1118 This seems to be a common mistake with adding new compartments: by default all the molecules are set to "is present" in the MSV building block, but especially all the molecules related to FcRn binding need to be unchecked in all new organs. Examples can be found in the trainings (especially "Antibody Therapy in Oncology") Stephan referred to.
